README.md

```diff
@@ -110,6 +110,28 @@ If you're using Botfront in a production environment, consider adding your own d
 **Important**
 If you are using the provided MongoDB deployment, `mongodb.mongodbHost` must follow the following pattern: `<release-name>-mongodb-service.<namespace>`
 
+
+### Enable and configure redis
+
+Redis is a data structure store, used by rasa as a lockstore. It should be enabled when working with multiple rasa instances. As it ensure that
```

Comment:
We should also give an example of the lock store config to paragraph to add to endpoints to make it work.

Reply:
The docs have been updated, now they describe bit more the lockstore and the sticky session mechanism